code/game/mecha/makeshift/lockermech.dm

```diff
@@ -0,0 +1,55 @@
+/obj/mecha/makeshift
+	desc = "A locker with stolen wires, struts, electronics and airlock servos crudely assembled into something that ressembles the functions of a mech."
+	name = "Locker Mech"
+	icon_state = "lockermech"
+	health = 200 //same health as ripley but without any of the armor/deflections making it very vulnerable
+	deflect_chance = 0
+	lights_power = 5
+	step_in = 5 //Slower than ripley
+	max_temperature = 5000
+	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0, fire = 0, acid = 0)
```

Comment:
we don't have fire or acid armor.

Blatant copy-pasting will get you ever time.